src/dfcx_scrapi/core/entity_types.py

```diff
@@ -77,8 +77,9 @@ def entity_type_proto_to_dataframe(
                 entity_type_dict["entity_value"] = entity.value
                 for synonym in entity.synonyms:
                     entity_type_dict["synonyms"] = synonym
-                    main_df = pd.concat(
-                        [main_df, entity_type_dict], ignore_index=True)
+                    entity_type_df = pd.DataFrame(entity_type_dict,index=[0])
```

Comment:
This is nit-picky, but please use whitespaces as needed to make sure we're following [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html) (3.6 Whitespace).